src/System.Net.Mail/tests/Functional/SmtpServer.cs

```diff
@@ -77,7 +77,8 @@ public void Run()
             }
         }
 
-        // return false == terminate
+        public bool Support_SMTPUTF8 { get; set; }
+		// return false == terminate
```

Comment:
It's a nitpick, but this line is slightly less indented than the rest of the code. It should line up with the code below.